From 89551f7c5865fafa039ea27e2e0ba662eb47d712 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Thu, 3 May 2018 11:16:05 +0800
Subject: [PATCH 3706/5242] MLK-18069 ARM: dts: imx7ulp-evk: correct the touch
 setting

commit  8690896ac5c5efe22e3bfbfbc0dfec37f2967f7d from
https://source.codeaurora.org/external/imx/linux-imx.git

The MIPI DSI config the DPI as 480 * 854, so correct the touch
display-coords property, to aligned with MIPI DSI.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 3c08414..eb1f8a2 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -412,14 +412,13 @@
 		focaltech,reset-gpio = <&gpio_ptf 1 0x01>;
 		focaltech,irq-gpio = <&gpio_ptf 0 0x02>;
 		focaltech,max-touch-number = <5>;
-		focaltech,display-coords =  <0 0 272 480>;
+		focaltech,display-coords =  <0 0 480 854>;
 
 		focaltech,have-key;
 		focaltech,key-number = <3>;
 		focaltech,keys = <139 102 158>;
 		focaltech,key-y-coord = <2000>;
 		focaltech,key-x-coords = <200 600 800>;
-		focaltech,scaling-down-half;
 	};
 };
 
-- 
1.7.9.5

